Makefile

```diff
@@ -9,4 +9,5 @@ install-deps:
 		pwsh -Command "& .\assets\scripts\win_deps.ps1"; \
 	else \
 		echo "Detected Unix/Linux. Skipping install..."; \
+		bash -ex assets/scripts/mac_deps.sh; \
```

Comment:
Cannot use OOTB since it depends on brew. Need to adapt for apt-get for linux. Also need to adjust the previous log to not say skipping install